pkg/monitoring/metrics/operator_metrics.go

```diff
@@ -0,0 +1,39 @@
+/*
+Copyright 2019 The hostpath provisioner operator Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package metrics
+
+import (
+	"github.com/machadovilaca/operator-observability/pkg/operatormetrics"
+)
+
+var (
+	operatorMetrics = []operatormetrics.Metric{
+		readyGauge,
+	}
+
+	readyGauge = operatormetrics.NewGauge(
+		operatormetrics.MetricOpts{
+			Name: "kubevirt_hpp_cr_ready",
+			Help: "The number of running hostpath-provisioner-operator pods",
```

Comment:
That help doesn't make sense, this is about if the HPP CR status reports ready or not. Not the number of hpp pods.

Reply:
You are right I replaced the help by mistake. 
@awels fixed now